packages/component-library/src/components/plmg-button/plmg-button.tsx

```diff
@@ -250,40 +278,44 @@ export class Button {
 
     if (this.href) {
       return (
-        <a
-          class={classes}
-          href={this.href}
-          rel={this.rel}
-          target={this.target}
-          aria-label={this.label}
-        >
+        <Host style={{ width: this.fullWidth ? 'full-width' : 'fit-content' }}>
+          <a
+            class={classes}
+            href={this.href}
+            rel={this.rel}
+            target={this.target}
+            aria-label={this.label}
+          >
+            {this.hasIconLeft() && (
+              <plmg-svg-icon class={'icon-left'} icon={this.iconLeft} />
+            )}
+            {this.hasIconCenter() && (
+              <plmg-svg-icon class={'icon-center'} icon={this.iconCenter} />
+            )}
+            <slot />
```

Comment:
```suggestion
            <slot></slot>
```
Safer to not use self-closing tags

Reply:
added